src/projectM-sdl/projectM_SDL_main.cpp

```diff
@@ -1,16 +1,35 @@
-//
-//  main.cpp
-//  projectM-sdl
-//
-//  Created by Mischa Spiegelmock on 6/3/15.
-//
-//  This is an implementation of projectM using libSDL2
+/**
+* projectM -- Milkdrop-esque visualisation SDK
+* Copyright (C)2003-2019 projectM Team
+*
+* This library is free software; you can redistribute it and/or
+* modify it under the terms of the GNU Lesser General Public
+* License as published by the Free Software Foundation; either
+* version 2.1 of the License, or (at your option) any later version.
+*
+* This library is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public
+* License along with this library; if not, write to the Free Software
+* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* See 'LICENSE.txt' included within this release
+*
+* projectM-sdl
+* This is an implementation of projectM using libSDL2
+* 
+* main.cpp
+* Authors: Created by Mischa Spiegelmock on 6/3/15.
+*
+* 
+* experimental Stereoscopic SBS driver functionality by
+*	RobertPancoast77@gmail.com
+*/
 
 #include "pmSDL.hpp"
 
-#define FAKE_AUDIO          0
```

Comment:
why remove? FAKE_AUDIO is at least useful for testing. is it documented anywhere else?

Reply:
I moved theses definitions to the header file so they can be effective across multiple files.